Doc/library/compileall.rst

```diff
@@ -56,12 +56,19 @@ compile Python sources.
    executed.
 
 .. option:: -s strip_prefix
+
+.. warning::
+   Removing the prefix makes paths relative to it.
+
 .. option:: -p prepend_prefix
 
    Remove (``-s``) or append (``-p``) the given prefix of paths
    recorded in the ``.pyc`` files.
    Cannot be combined with ``-d``.
 
+   ``-s`` and ``-p`` can be used simultaneously.
```

Comment:
Put this after `Cannot be combined with -d` and simply say "Can be used with -s" (for -p) and "Can be used with -p" (for -s).